controller/array_action/array_mediator_svc.py

```diff
@@ -846,6 +846,26 @@ def _get_detailed_hosts_list_cmd(self, host_list):
             writer.write(LIST_HOSTS_CMD_FORMAT.format(HOST_ID=host.id))
         return writer.getvalue()
 
+    def get_host_by_name(self, host_name):
+        cli_host_by_name = self._get_cli_host_by_name(host_name, not_found_error=False)
+        if cli_host_by_name is None:
+            raise array_errors.HostNotFoundError(host_name)
```

Comment:
why rasing the error [here](https://github.com/IBM/ibm-block-csi-driver/blob/9148a35a9501467ff03d3b5e6583b9f8d4907be1/controller/array_action/array_mediator_svc.py#L852) and not [here](https://github.com/IBM/ibm-block-csi-driver/blob/9148a35a9501467ff03d3b5e6583b9f8d4907be1/controller/array_action/array_mediator_svc.py#L1067)?